parser-typechecker/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -1,4 +1,5 @@
 {-# OPTIONS_GHC -fno-warn-partial-type-signatures #-}
+{-# OPTIONS_GHC -fwarn-incomplete-patterns #-}
```

Comment:
Uh oh, is this one that we want to keep?

Reply:
I stumbled upon the need for this due to the fact that `DefinitionsNotFound` was not being handled, I expected to get a compilation error, instead got an unhandled exception. Can remove it, but it feels like the sum type is too big to manoeuvre without it. I can remove it, not a problem.

Reply:
Oh, this adds warnings; I thought it was removing them.  

Correct me if I'm wrong, but this is already included in `-Wall`, which is turned on in the .cabal file.  Did adding the flag again actually do anything?

Reply:
As per my original comment, if I am adding a new constructor in Output, the "turn warnings on" is needed, as otherwise the code compiles and ends up in a runtime error (unhandled exception). I only added it cause I got the exception in tests.

Reply:
Right, but I'm asking whether you observed a difference after adding the flag?  Or did you added it to be safe, for next time?

(If you observed a difference after adding the flag, then it means something is broken with the build file.  If you didn't observe a difference after adding the flag, then it's safe to delete.)

Reply:
Sorry, should have clarified. Upon adding the flag, the ghc immediately gave me an "this case is unhandled error".

Reply:
Should be reproducable on master I would expect, just remove 1-2 cases and try to compile.